tests/assets/js/vendor/mootools-1.6.0.js

```diff
@@ -2098,6 +2098,12 @@ local.setDocument = function(document){
 				brokenSecondClassNameGEBCN = (testNode.getElementsByClassName('a').length != 2);
 			} catch (e){}
 
+			// Opera 9.6 getElementsByClassName doesnt detects the class if its not the first one
```

Comment:
This file is still showing as modified.

Reply:
looks like sourcetree inserted it twice when it "reverted" for some reason.  thanks for catching